example/node/lcov.info

```diff
@@ -1,107 +0,0 @@
-TN:
-SF:/home/ryanluker/dev/vscode-coverage-gutters/example/node/test.js
```

Comment:
These absolute paths are wrong on every machine but @ryanluker's, so we should not rely on them.

Reply:
these paths are fine in the new way we do path comparison (now that top score is dead). it also has the benefit of the build machine not needed to be able to generate xml for node, python, php, etc.